packages/core/src/main/start-main-application/lens-window/application-window/session-certificate-verifier.injectable.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { timingSafeEqual, X509Certificate } from "crypto";
+import type { Request } from "electron";
+import lensProxyCertificateInjectable from "../../../../common/certificate/lens-proxy-certificate.injectable";
+
+// see https://www.electronjs.org/docs/latest/api/session#sessetcertificateverifyprocproc
+export enum ChromiumNetError {
+  SUCCESS = 0,
+  FAILURE = -2,
+  RESULT_FROM_CHROMIUM = -3,
+}
```

Comment:
I wonder if there's an official `enum` for this from eg. chromium itself.

Reply:
I was wondering the same, feels bad to include error codes here from chromium... but I could not find anything.